pkg/cmd/gist/create/create.go

```diff
@@ -154,3 +172,23 @@ func processFiles(stdin io.ReadCloser, filenames []string) (map[string]string, e
 
 	return fs, nil
 }
+
+func guessGistName(files map[string]string) (string, error) {
+	filenames := make([]string, 0, len(files))
+	gistName := ""
+
+	re := regexp.MustCompile(`^gistfile\d+\.txt$`)
+	for k := range files {
+		match := re.MatchString(k)
+		if !match {
```

Comment:
Nit: these two can be collapsed into the same line. There's not much we gain by assigning a variable here, since the original expression is easily understandable